src/NotificationTemplate.php

```diff
@@ -437,6 +431,21 @@ public static function process($string, $data)
         return $string;
     }
 
+    public static function convertRelativeToAbsolute($string)
```

Comment:
For every new method, try to use type hinting.
```suggestion
    public static function convertRelativeToAbsolute(string $string): string
```